src/lib/y2network/connection_config/wireless.rb

```diff
@@ -113,6 +114,28 @@ def mode=(wireless_mode)
       def keys?
         !(keys || []).compact.all?(&:empty?)
       end
+
+      # Return a clone of this object with all sensitive fields (passwords etc.) obscured.
+      # Use this for log output to avoid leaking passwords.
+      def sanitized
+        san = dup
+        san.wpa_psk = sanitized_field(san.wpa_psk)
+        san.wpa_password = sanitized_field(san.wpa_password)
+        san.client_key_password = sanitized_field(san.client_key_password)
```

Comment:
Consider adding also **_wep_keys_** to the list of attributes to be concealed.

Reply:
yup, done in the new one #1360.